DOCSUP-10513: Add MaterializeMySQL Engine column comments support (#26458)
ClickHouse/ClickHouse: docs/en/engines/database-engines/materialize-mysql.md

Creates ClickHouse database with all the tables existing in MySQL, and all the data in those tables.

> ClickHouse server works as MySQL replica. It reads binlog and performs DDL and DML queries. ClickHouse database created with MaterializeMySQL contains all column comments from the MySQL database that materialized.

Reviewer: This info about column comments is not as important to be in this section. Enough to add to `### Selecting from MaterializeMySQL Tables {#select}`.

Author: Ok, i'll remove it ;)

- If `_sign` is not specified in the `SELECT` query, `WHERE _sign=1` is used by default. So the deleted rows are not included into the result set.

> - The result includes columns comments from the MySQL database that materialized (if they exist).

Reviewer: ```suggestion
- The result includes columns comments in case they exist in MySQL database tables.
```
